chore(root): publish modules

pranavjain97 opened the pull request:
- @bitgo/abstract-cosmos@11.13.1
 - @bitgo/abstract-eth@24.11.1
 - @bitgo/abstract-lightning@6.4.0
 - @bitgo/abstract-substrate@1.10.1
 - @bitgo/abstract-utxo@9.25.0
 - @bitgo/account-lib@27.10.1
 - bitgo@49.6.0
 - @bitgo/express@14.5.0
 - @bitgo/key-card@0.27.1
 - @bitgo/sdk-api@1.68.0
 - @bitgo/sdk-coin-ada@4.13.1
 - @bitgo/sdk-coin-algo@2.4.1
 - @bitgo/sdk-coin-apechain@1.2.1
 - @bitgo/sdk-coin-apt@2.5.1
 - @bitgo/sdk-coin-arbeth@21.8.1
 - @bitgo/sdk-coin-asi@1.4.1
 - @bitgo/sdk-coin-atom@13.7.1
 - @bitgo/sdk-coin-avaxc@6.3.1
 - @bitgo/sdk-coin-avaxp@5.3.1
 - @bitgo/sdk-coin-baby@1.7.1
 - @bitgo/sdk-coin-bch@2.3.1
 - @bitgo/sdk-coin-bcha@2.4.1
 - @bitgo/sdk-coin-bera@2.5.1
 - @bitgo/sdk-coin-bld@3.3.1
 - @bitgo/sdk-coin-bsc@22.6.1
 - @bitgo/sdk-coin-bsv@2.3.1
 - @bitgo/sdk-coin-btc@2.7.1
 - @bitgo/sdk-coin-btg@2.3.1
 - @bitgo/sdk-coin-celo@5.2.1
 - @bitgo/sdk-coin-coredao@2.5.1
 - @bitgo/sdk-coin-coreum@21.3.1
 - @bitgo/sdk-coin-cosmos@1.5.1
 - @bitgo/sdk-coin-cronos@1.5.1
 - @bitgo/sdk-coin-cspr@2.3.1
 - @bitgo/sdk-coin-dash@2.3.1
 - @bitgo/sdk-coin-doge@2.3.1
 - @bitgo/sdk-coin-dot@4.4.1
 - @bitgo/sdk-coin-eos@3.4.1
 - @bitgo/sdk-coin-etc@2.4.1
 - @bitgo/sdk-coin-eth@25.2.1
 - @bitgo/sdk-coin-ethlike@2.1.1
 - @bitgo/sdk-coin-ethw@20.2.1
 - @bitgo/sdk-coin-evm@1.6.1
 - @bitgo/sdk-coin-flr@1.5.1
 - @bitgo/sdk-coin-hash@3.4.1
 - @bitgo/sdk-coin-hbar@2.3.1
 - @bitgo/sdk-coin-icp@1.18.1
 - @bitgo/sdk-coin-initia@2.3.1
 - @bitgo/sdk-coin-injective@3.3.1
 - @bitgo/sdk-coin-iota@1.1.0
 - @bitgo/sdk-coin-islm@2.3.1
 - @bitgo/sdk-coin-lnbtc@1.4.1
 - @bitgo/sdk-coin-ltc@3.3.1
 - @bitgo/sdk-coin-mantra@1.2.1
 - @bitgo/sdk-coin-mon@1.3.1
 - @bitgo/sdk-coin-near@2.10.1
 - @bitgo/sdk-coin-oas@2.4.1
 - @bitgo/sdk-coin-opeth@18.6.1
 - @bitgo/sdk-coin-osmo@3.3.1
 - @bitgo/sdk-coin-polygon@21.4.1
 - @bitgo/sdk-coin-polyx@1.9.0
 - @bitgo/sdk-coin-rbtc@2.2.1
 - @bitgo/sdk-coin-rune@1.4.1
 - @bitgo/sdk-coin-sei@3.3.1
 - @bitgo/sdk-coin-sgb@1.5.1
 - @bitgo/sdk-coin-sol@6.1.1
 - @bitgo/sdk-coin-soneium@1.7.1
 - @bitgo/sdk-coin-stt@1.3.1
 - @bitgo/sdk-coin-stx@3.9.1
 - @bitgo/sdk-coin-sui@5.18.1
 - @bitgo/sdk-coin-tao@1.11.1
 - @bitgo/sdk-coin-tia@3.3.1
 - @bitgo/sdk-coin-ton@3.8.1
 - @bitgo/sdk-coin-trx@3.5.1
 - @bitgo/sdk-coin-vet@2.5.0
 - @bitgo/sdk-coin-wemix@1.4.1
 - @bitgo/sdk-coin-world@1.5.1
 - @bitgo/sdk-coin-xdc@1.4.1
 - @bitgo/sdk-coin-xlm@3.5.1
 - @bitgo/sdk-coin-xrp@3.10.1
 - @bitgo/sdk-coin-xtz@2.7.1
 - @bitgo/sdk-coin-zec@2.3.1
 - @bitgo/sdk-coin-zeta@3.3.1
 - @bitgo/sdk-coin-zketh@2.3.1
 - @bitgo/sdk-core@36.6.0
 - @bitgo/sdk-rpc-wrapper@2.2.1
 - @bitgo/sdk-test@9.0.6
 - @bitgo/statics@57.6.0
 - @bitgo/utxo-bin@3.9.1
 - @bitgo/utxo-core@1.18.0
 - @bitgo/utxo-ord@1.21.1
 - @bitgo/utxo-staking@1.21.1
 - @bitgo/web-demo@3.2.8

<!--
# Please be aware of the following when making your pull request:

## Description

Please include a summary of your proposed changes and which issue is being addressed. Please also include relevant motivation and context. List any dependencies that are required for this change.

## Issue Number

Internal Users - Please include the related internal tracking number (e.g. BG-000000).
External Users - Please link to any relevant github issues as necessary.

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration.

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] My code compiles correctly for both Node and Browser environments
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] My commits follow [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) and I have properly described any BREAKING CHANGES
- [ ] The ticket or github issue was included in the commit message as a reference
- [ ] I have made corresponding changes to the documentation and on any new/updated functions and/or methods - [jsdoc](https://jsdoc.app/)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
-->

pranavjain97 committed:
chore(root): publish modules

 - @bitgo/abstract-cosmos@11.13.1
 - @bitgo/abstract-eth@24.11.1
 - @bitgo/abstract-lightning@6.4.0
 - @bitgo/abstract-substrate@1.10.1
 - @bitgo/abstract-utxo@9.25.0
 - @bitgo/account-lib@27.10.1
 - bitgo@49.6.0
 - @bitgo/express@14.5.0
 - @bitgo/key-card@0.27.1
 - @bitgo/sdk-api@1.68.0
 - @bitgo/sdk-coin-ada@4.13.1
 - @bitgo/sdk-coin-algo@2.4.1
 - @bitgo/sdk-coin-apechain@1.2.1
 - @bitgo/sdk-coin-apt@2.5.1
 - @bitgo/sdk-coin-arbeth@21.8.1
 - @bitgo/sdk-coin-asi@1.4.1
 - @bitgo/sdk-coin-atom@13.7.1
 - @bitgo/sdk-coin-avaxc@6.3.1
 - @bitgo/sdk-coin-avaxp@5.3.1
 - @bitgo/sdk-coin-baby@1.7.1
 - @bitgo/sdk-coin-bch@2.3.1
 - @bitgo/sdk-coin-bcha@2.4.1
 - @bitgo/sdk-coin-bera@2.5.1
 - @bitgo/sdk-coin-bld@3.3.1
 - @bitgo/sdk-coin-bsc@22.6.1
 - @bitgo/sdk-coin-bsv@2.3.1
 - @bitgo/sdk-coin-btc@2.7.1
 - @bitgo/sdk-coin-btg@2.3.1
 - @bitgo/sdk-coin-celo@5.2.1
 - @bitgo/sdk-coin-coredao@2.5.1
 - @bitgo/sdk-coin-coreum@21.3.1
 - @bitgo/sdk-coin-cosmos@1.5.1
 - @bitgo/sdk-coin-cronos@1.5.1
 - @bitgo/sdk-coin-cspr@2.3.1
 - @bitgo/sdk-coin-dash@2.3.1
 - @bitgo/sdk-coin-doge@2.3.1
 - @bitgo/sdk-coin-dot@4.4.1
 - @bitgo/sdk-coin-eos@3.4.1
 - @bitgo/sdk-coin-etc@2.4.1
 - @bitgo/sdk-coin-eth@25.2.1
 - @bitgo/sdk-coin-ethlike@2.1.1
 - @bitgo/sdk-coin-ethw@20.2.1
 - @bitgo/sdk-coin-evm@1.6.1
 - @bitgo/sdk-coin-flr@1.5.1
 - @bitgo/sdk-coin-hash@3.4.1
 - @bitgo/sdk-coin-hbar@2.3.1
 - @bitgo/sdk-coin-icp@1.18.1
 - @bitgo/sdk-coin-initia@2.3.1
 - @bitgo/sdk-coin-injective@3.3.1
 - @bitgo/sdk-coin-iota@1.1.0
 - @bitgo/sdk-coin-islm@2.3.1
 - @bitgo/sdk-coin-lnbtc@1.4.1
 - @bitgo/sdk-coin-ltc@3.3.1
 - @bitgo/sdk-coin-mantra@1.2.1
 - @bitgo/sdk-coin-mon@1.3.1
 - @bitgo/sdk-coin-near@2.10.1
 - @bitgo/sdk-coin-oas@2.4.1
 - @bitgo/sdk-coin-opeth@18.6.1
 - @bitgo/sdk-coin-osmo@3.3.1
 - @bitgo/sdk-coin-polygon@21.4.1
 - @bitgo/sdk-coin-polyx@1.9.0
 - @bitgo/sdk-coin-rbtc@2.2.1
 - @bitgo/sdk-coin-rune@1.4.1
 - @bitgo/sdk-coin-sei@3.3.1
 - @bitgo/sdk-coin-sgb@1.5.1
 - @bitgo/sdk-coin-sol@6.1.1
 - @bitgo/sdk-coin-soneium@1.7.1
 - @bitgo/sdk-coin-stt@1.3.1
 - @bitgo/sdk-coin-stx@3.9.1
 - @bitgo/sdk-coin-sui@5.18.1
 - @bitgo/sdk-coin-tao@1.11.1
 - @bitgo/sdk-coin-tia@3.3.1
 - @bitgo/sdk-coin-ton@3.8.1
 - @bitgo/sdk-coin-trx@3.5.1
 - @bitgo/sdk-coin-vet@2.5.0
 - @bitgo/sdk-coin-wemix@1.4.1
 - @bitgo/sdk-coin-world@1.5.1
 - @bitgo/sdk-coin-xdc@1.4.1
 - @bitgo/sdk-coin-xlm@3.5.1
 - @bitgo/sdk-coin-xrp@3.10.1
 - @bitgo/sdk-coin-xtz@2.7.1
 - @bitgo/sdk-coin-zec@2.3.1
 - @bitgo/sdk-coin-zeta@3.3.1
 - @bitgo/sdk-coin-zketh@2.3.1
 - @bitgo/sdk-core@36.6.0
 - @bitgo/sdk-rpc-wrapper@2.2.1
 - @bitgo/sdk-test@9.0.6
 - @bitgo/statics@57.6.0
 - @bitgo/utxo-bin@3.9.1
 - @bitgo/utxo-core@1.18.0
 - @bitgo/utxo-ord@1.21.1
 - @bitgo/utxo-staking@1.21.1
 - @bitgo/web-demo@3.2.8

margueriteblair reviewed (commented):
express changes?

mohammadalfaiyazbitgo committed:
chore(root): release express

TICKET: WP-534

margueriteblair reviewed (commented):
sdk tests failing :'( can approve once those are remedied

zahin-mohammad committed:
Merge branch 'master' into rel/latest

zahin-mohammad committed:
chore(root): publish modules

 - @bitgo/abstract-cosmos@11.14.0
 - @bitgo/abstract-eth@24.11.2
 - @bitgo/abstract-lightning@6.5.0
 - @bitgo/abstract-substrate@1.10.2
 - @bitgo/abstract-utxo@9.25.1
 - @bitgo/account-lib@27.10.2
 - @bitgo/babylonlabs-io-btc-staking-ts@2.4.1
 - bitgo@49.6.1
 - @bitgo/express@14.6.0
 - @bitgo/key-card@0.27.2
 - @bitgo/sdk-api@1.68.1
 - @bitgo/sdk-coin-ada@4.13.2
 - @bitgo/sdk-coin-algo@2.4.2
 - @bitgo/sdk-coin-apechain@1.2.2
 - @bitgo/sdk-coin-apt@2.5.2
 - @bitgo/sdk-coin-arbeth@21.8.2
 - @bitgo/sdk-coin-asi@1.4.2
 - @bitgo/sdk-coin-atom@13.8.0
 - @bitgo/sdk-coin-avaxc@6.3.2
 - @bitgo/sdk-coin-avaxp@5.3.2
 - @bitgo/sdk-coin-baby@1.7.2
 - @bitgo/sdk-coin-bch@2.3.2
 - @bitgo/sdk-coin-bcha@2.4.2
 - @bitgo/sdk-coin-bera@2.5.2
 - @bitgo/sdk-coin-bld@3.4.0
 - @bitgo/sdk-coin-bsc@22.7.0
 - @bitgo/sdk-coin-bsv@2.3.2
 - @bitgo/sdk-coin-btc@2.7.2
 - @bitgo/sdk-coin-btg@2.3.2
 - @bitgo/sdk-coin-celo@5.2.2
 - @bitgo/sdk-coin-coredao@2.5.2
 - @bitgo/sdk-coin-coreum@21.4.0
 - @bitgo/sdk-coin-cosmos@1.5.2
 - @bitgo/sdk-coin-cronos@1.5.2
 - @bitgo/sdk-coin-cspr@2.3.2
 - @bitgo/sdk-coin-dash@2.3.2
 - @bitgo/sdk-coin-doge@2.3.2
 - @bitgo/sdk-coin-dot@4.4.2
 - @bitgo/sdk-coin-eos@3.4.2
 - @bitgo/sdk-coin-etc@2.4.2
 - @bitgo/sdk-coin-eth@25.2.2
 - @bitgo/sdk-coin-ethlike@2.1.2
 - @bitgo/sdk-coin-ethw@20.2.2
 - @bitgo/sdk-coin-evm@1.6.2
 - @bitgo/sdk-coin-flr@1.5.2
 - @bitgo/sdk-coin-hash@3.5.0
 - @bitgo/sdk-coin-hbar@2.3.2
 - @bitgo/sdk-coin-icp@1.18.2
 - @bitgo/sdk-coin-initia@2.3.2
 - @bitgo/sdk-coin-injective@3.4.0
 - @bitgo/sdk-coin-iota@1.1.1
 - @bitgo/sdk-coin-islm@2.3.2
 - @bitgo/sdk-coin-lnbtc@1.4.2
 - @bitgo/sdk-coin-ltc@3.3.2
 - @bitgo/sdk-coin-mantra@1.2.2
 - @bitgo/sdk-coin-mon@1.3.2
 - @bitgo/sdk-coin-near@2.10.2
 - @bitgo/sdk-coin-oas@2.4.2
 - @bitgo/sdk-coin-opeth@18.6.2
 - @bitgo/sdk-coin-osmo@3.4.0
 - @bitgo/sdk-coin-polygon@21.4.2
 - @bitgo/sdk-coin-polyx@1.9.1
 - @bitgo/sdk-coin-rbtc@2.2.2
 - @bitgo/sdk-coin-rune@1.5.0
 - @bitgo/sdk-coin-sei@3.4.0
 - @bitgo/sdk-coin-sgb@1.5.2
 - @bitgo/sdk-coin-sol@6.1.2
 - @bitgo/sdk-coin-soneium@1.7.2
 - @bitgo/sdk-coin-stt@1.3.2
 - @bitgo/sdk-coin-stx@3.9.2
 - @bitgo/sdk-coin-sui@5.18.2
 - @bitgo/sdk-coin-tao@1.11.2
 - @bitgo/sdk-coin-tia@3.4.0
 - @bitgo/sdk-coin-ton@3.8.2
 - @bitgo/sdk-coin-trx@3.5.2
 - @bitgo/sdk-coin-vet@2.5.1
 - @bitgo/sdk-coin-wemix@1.4.2
 - @bitgo/sdk-coin-world@1.5.2
 - @bitgo/sdk-coin-xdc@1.4.2
 - @bitgo/sdk-coin-xlm@3.5.2
 - @bitgo/sdk-coin-xrp@3.10.2
 - @bitgo/sdk-coin-xtz@2.7.2
 - @bitgo/sdk-coin-zec@2.3.2
 - @bitgo/sdk-coin-zeta@3.4.0
 - @bitgo/sdk-coin-zketh@2.3.2
 - @bitgo/sdk-core@36.6.1
 - @bitgo/sdk-rpc-wrapper@2.2.2
 - @bitgo/sdk-test@9.0.7
 - @bitgo/statics@57.7.0
 - @bitgo/utxo-bin@3.9.2
 - @bitgo/utxo-ord@1.21.2
 - @bitgo/utxo-staking@1.21.2
 - @bitgo/web-demo@3.2.9

zahin-mohammad committed:
chore(root): release express

TICKET: WP-0000

zahin-mohammad committed:
Merge branch 'master' into rel/latest

zahin-mohammad committed:
chore(root): publish modules

 - @bitgo/abstract-cosmos@11.14.1
 - @bitgo/abstract-eth@24.11.3
 - @bitgo/abstract-lightning@6.5.1
 - @bitgo/abstract-substrate@1.10.3
 - @bitgo/abstract-utxo@9.25.2
 - @bitgo/account-lib@27.10.3
 - bitgo@49.7.0
 - @bitgo/express@14.6.1
 - @bitgo/key-card@0.27.3
 - @bitgo/sdk-api@1.68.2
 - @bitgo/sdk-coin-ada@4.13.3
 - @bitgo/sdk-coin-algo@2.4.3
 - @bitgo/sdk-coin-apechain@1.2.3
 - @bitgo/sdk-coin-apt@2.5.3
 - @bitgo/sdk-coin-arbeth@21.8.3
 - @bitgo/sdk-coin-asi@1.4.3
 - @bitgo/sdk-coin-atom@13.8.1
 - @bitgo/sdk-coin-avaxc@6.3.3
 - @bitgo/sdk-coin-avaxp@5.3.3
 - @bitgo/sdk-coin-baby@1.7.3
 - @bitgo/sdk-coin-bch@2.3.3
 - @bitgo/sdk-coin-bcha@2.4.3
 - @bitgo/sdk-coin-bera@2.5.3
 - @bitgo/sdk-coin-bld@3.4.1
 - @bitgo/sdk-coin-bsc@22.7.1
 - @bitgo/sdk-coin-bsv@2.3.3
 - @bitgo/sdk-coin-btc@2.7.3
 - @bitgo/sdk-coin-btg@2.3.3
 - @bitgo/sdk-coin-celo@5.2.3
 - @bitgo/sdk-coin-coredao@2.5.3
 - @bitgo/sdk-coin-coreum@21.4.1
 - @bitgo/sdk-coin-cosmos@1.5.3
 - @bitgo/sdk-coin-cronos@1.5.3
 - @bitgo/sdk-coin-cspr@2.3.3
 - @bitgo/sdk-coin-dash@2.3.3
 - @bitgo/sdk-coin-doge@2.3.3
 - @bitgo/sdk-coin-dot@4.4.3
 - @bitgo/sdk-coin-eos@3.4.3
 - @bitgo/sdk-coin-etc@2.4.3
 - @bitgo/sdk-coin-eth@25.2.3
 - @bitgo/sdk-coin-ethlike@2.1.3
 - @bitgo/sdk-coin-ethw@20.2.3
 - @bitgo/sdk-coin-evm@1.6.3
 - @bitgo/sdk-coin-flr@1.5.3
 - @bitgo/sdk-coin-hash@3.5.1
 - @bitgo/sdk-coin-hbar@2.3.3
 - @bitgo/sdk-coin-icp@1.18.3
 - @bitgo/sdk-coin-initia@2.3.3
 - @bitgo/sdk-coin-injective@3.4.1
 - @bitgo/sdk-coin-iota@1.1.2
 - @bitgo/sdk-coin-islm@2.3.3
 - @bitgo/sdk-coin-lnbtc@1.4.3
 - @bitgo/sdk-coin-ltc@3.3.3
 - @bitgo/sdk-coin-mantra@1.2.3
 - @bitgo/sdk-coin-mon@1.3.3
 - @bitgo/sdk-coin-near@2.10.3
 - @bitgo/sdk-coin-oas@2.4.3
 - @bitgo/sdk-coin-opeth@18.6.3
 - @bitgo/sdk-coin-osmo@3.4.1
 - @bitgo/sdk-coin-polygon@21.4.3
 - @bitgo/sdk-coin-polyx@1.9.2
 - @bitgo/sdk-coin-rbtc@2.2.3
 - @bitgo/sdk-coin-rune@1.5.1
 - @bitgo/sdk-coin-sei@3.4.1
 - @bitgo/sdk-coin-sgb@1.5.3
 - @bitgo/sdk-coin-sol@6.1.3
 - @bitgo/sdk-coin-soneium@1.7.3
 - @bitgo/sdk-coin-stt@1.3.3
 - @bitgo/sdk-coin-stx@3.9.3
 - @bitgo/sdk-coin-sui@5.18.3
 - @bitgo/sdk-coin-tao@1.11.3
 - @bitgo/sdk-coin-tia@3.4.1
 - @bitgo/sdk-coin-ton@3.8.3
 - @bitgo/sdk-coin-trx@3.5.3
 - @bitgo/sdk-coin-vet@2.5.2
 - @bitgo/sdk-coin-wemix@1.4.3
 - @bitgo/sdk-coin-world@1.5.3
 - @bitgo/sdk-coin-xdc@1.4.3
 - @bitgo/sdk-coin-xlm@3.5.3
 - @bitgo/sdk-coin-xrp@3.10.3
 - @bitgo/sdk-coin-xtz@2.7.3
 - @bitgo/sdk-coin-zec@2.3.3
 - @bitgo/sdk-coin-zeta@3.4.1
 - @bitgo/sdk-coin-zketh@2.3.3
 - @bitgo/sdk-core@36.7.0
 - @bitgo/sdk-rpc-wrapper@2.2.3
 - @bitgo/sdk-test@9.0.8
 - @bitgo/utxo-ord@1.21.3
 - @bitgo/web-demo@3.2.10

zahin-mohammad committed:
chore(root): release express

TICKET: WP-0000

pranavjain97 committed:
Merge branch 'master' into rel/latest

pranavjain97 committed:
fix(sdk-coin-iota): fix iota packages from merge conflict

Ticket: WP-0000

pranavjain97 committed:
chore(root): publish modules

 - @bitgo/abstract-cosmos@11.14.2
 - @bitgo/abstract-eth@24.12.0
 - @bitgo/abstract-lightning@7.0.0
 - @bitgo/abstract-substrate@1.10.4
 - @bitgo/abstract-utxo@9.26.0
 - @bitgo/account-lib@27.10.4
 - bitgo@50.0.0
 - @bitgo/express@15.0.0
 - @bitgo/key-card@0.27.4
 - @bitgo/sdk-api@1.68.3
 - @bitgo/sdk-coin-ada@4.14.0
 - @bitgo/sdk-coin-algo@2.4.4
 - @bitgo/sdk-coin-apechain@1.2.4
 - @bitgo/sdk-coin-apt@2.5.4
 - @bitgo/sdk-coin-arbeth@21.8.4
 - @bitgo/sdk-coin-asi@1.4.4
 - @bitgo/sdk-coin-atom@13.8.2
 - @bitgo/sdk-coin-avaxc@6.3.4
 - @bitgo/sdk-coin-avaxp@5.3.4
 - @bitgo/sdk-coin-baby@1.7.4
 - @bitgo/sdk-coin-bch@2.3.4
 - @bitgo/sdk-coin-bcha@2.4.4
 - @bitgo/sdk-coin-bera@2.5.4
 - @bitgo/sdk-coin-bld@3.4.2
 - @bitgo/sdk-coin-bsc@22.7.2
 - @bitgo/sdk-coin-bsv@2.3.4
 - @bitgo/sdk-coin-btc@2.7.4
 - @bitgo/sdk-coin-btg@2.3.4
 - @bitgo/sdk-coin-celo@5.2.4
 - @bitgo/sdk-coin-coredao@2.5.4
 - @bitgo/sdk-coin-coreum@21.4.2
 - @bitgo/sdk-coin-cosmos@1.5.4
 - @bitgo/sdk-coin-cronos@1.5.4
 - @bitgo/sdk-coin-cspr@2.3.4
 - @bitgo/sdk-coin-dash@2.3.4
 - @bitgo/sdk-coin-doge@2.3.4
 - @bitgo/sdk-coin-dot@4.4.4
 - @bitgo/sdk-coin-eos@3.4.4
 - @bitgo/sdk-coin-etc@2.4.4
 - @bitgo/sdk-coin-eth@25.2.4
 - @bitgo/sdk-coin-ethlike@2.1.4
 - @bitgo/sdk-coin-ethw@20.2.4
 - @bitgo/sdk-coin-evm@1.6.4
 - @bitgo/sdk-coin-flr@1.5.4
 - @bitgo/sdk-coin-hash@3.5.2
 - @bitgo/sdk-coin-hbar@2.3.4
 - @bitgo/sdk-coin-icp@1.18.4
 - @bitgo/sdk-coin-initia@2.3.4
 - @bitgo/sdk-coin-injective@3.4.2
 - @bitgo/sdk-coin-iota@1.2.0
 - @bitgo/sdk-coin-islm@2.3.4
 - @bitgo/sdk-coin-lnbtc@1.4.4
 - @bitgo/sdk-coin-ltc@3.3.4
 - @bitgo/sdk-coin-mantra@1.2.4
 - @bitgo/sdk-coin-mon@1.3.4
 - @bitgo/sdk-coin-near@2.10.4
 - @bitgo/sdk-coin-oas@2.4.4
 - @bitgo/sdk-coin-opeth@18.6.4
 - @bitgo/sdk-coin-osmo@3.4.2
 - @bitgo/sdk-coin-polygon@21.4.4
 - @bitgo/sdk-coin-polyx@1.9.3
 - @bitgo/sdk-coin-rbtc@2.2.4
 - @bitgo/sdk-coin-rune@1.5.2
 - @bitgo/sdk-coin-sei@3.4.2
 - @bitgo/sdk-coin-sgb@1.5.4
 - @bitgo/sdk-coin-sol@6.1.4
 - @bitgo/sdk-coin-soneium@1.7.4
 - @bitgo/sdk-coin-stt@1.3.4
 - @bitgo/sdk-coin-stx@3.9.4
 - @bitgo/sdk-coin-sui@5.18.4
 - @bitgo/sdk-coin-tao@1.11.4
 - @bitgo/sdk-coin-tia@3.4.2
 - @bitgo/sdk-coin-ton@3.8.4
 - @bitgo/sdk-coin-trx@3.5.4
 - @bitgo/sdk-coin-vet@2.5.3
 - @bitgo/sdk-coin-wemix@1.4.4
 - @bitgo/sdk-coin-world@1.5.4
 - @bitgo/sdk-coin-xdc@1.4.4
 - @bitgo/sdk-coin-xlm@3.5.4
 - @bitgo/sdk-coin-xrp@3.10.4
 - @bitgo/sdk-coin-xtz@2.7.4
 - @bitgo/sdk-coin-zec@2.3.4
 - @bitgo/sdk-coin-zeta@3.4.2
 - @bitgo/sdk-coin-zketh@2.3.4
 - @bitgo/sdk-core@36.8.0
 - @bitgo/sdk-rpc-wrapper@2.2.4
 - @bitgo/sdk-test@9.0.9
 - @bitgo/statics@57.8.0
 - @bitgo/utxo-bin@3.9.3
 - @bitgo/utxo-core@1.19.0
 - @bitgo/utxo-ord@1.21.4
 - @bitgo/utxo-staking@1.21.3
 - @bitgo/web-demo@3.2.11

pranavjain97 committed:
chore(root): release express

Ticket: WP-0000
